### PR TITLE
Add manual page for opensc.conf

### PR DESCRIPTION
**Pull request by frankmorgner**
... allows distributing a minimal standard configuration

### CODE REVIEW

File: doc/files/opensc.conf.5.xml.in [hunk 1]
```diff
@@ -0,0 +1,1630 @@
...
+				<varlistentry>
+					<term>
+						<option>builtin_emulators = <replaceable>emulators</replaceable>;</option>
+					</term>
+					<listitem><para>
+							List of the builtin pkcs15 emulators to test
+							(Default: <literal>esteid, openpgp, tcos,
+								starcert, itacns, infocamere, postecert,
+								actalis, atrust-acos, gemsafeGPK,
+								gemsafeV1, tccardos, PIV-II</literal>)
```
Comment (Jakuje) [inline doc/files/opensc.conf.5.xml.in]: This list is probably outdated. There is much more emulators these days including the cac and coolkey.

File: doc/files/opensc.conf.5.xml.in [hunk 2]
```diff
@@ -0,0 +1,1630 @@
...
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>pkcs11_enable_InitToken = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Enable initialization and card recognition in
```
Comment (Jakuje) [inline doc/files/opensc.conf.5.xml.in]: I would put it as "Enable empty card recognition and initialization" to make this configuration option less cryptic.

File: doc/files/opensc.conf.5.xml.in [hunk 3]
```diff
@@ -0,0 +1,1630 @@
...
+										local)
+								</para></listitem>
+							</itemizedlist>
+						</para>
+						<para>
+							Card can contain more then one PINs or more then
+							one on-card application with its own PINs.
+							Normally, to access all of them with the PKCS#11
+							API a slot has to be created for all of them. Many
+							slots could be ennoying for some of widely used
```
Comment (Jakuje) [inline doc/files/opensc.conf.5.xml.in]: typo: `annoying`

File: doc/files/opensc.conf.5.xml.in [hunk 4]
```diff
@@ -0,0 +1,1630 @@
...
+							<option>create_slots_for_pins = "user";</option>
+					</para></listitem>
+				</varlistentry>
+			</variablelist>
+		</refsect2>
+
+	</refsect1>
+
+	<refsect1>
+		<title>Environment</title>
```
Comment (Jakuje) [inline doc/files/opensc.conf.5.xml.in]: For completeness, I would also list the `OPENSC_DRIVER` and `OPENSC_DEBUG` env variables.

**Comment by frankmorgner** [comment]
@marschap do you have any idea how to remove the newline after a `listitem` in an `itemizedlist`?

**Comment by frankmorgner** [comment]
I've integrated your comments, thanks @Jakuje

**Comment by marschap** [comment]
@frankmorgner: I am not aware of such a possibility. What's the exact issue you want to solve?

**Comment by frankmorgner** [comment]
@marschap look at the enumeration at the start of the manual page:

```
DESCRIPTION
       OpenSC obtains configuration data from the following sources in the following order

        1. command-line options

        2. environment variables

        3. Windows registry (if available)

        4. system-wide configuration file (/home/fm/.local/etc/opensc.conf)
```
What I'd like to get rid of are the empty lines within the enumeration... The code for the above sniplet looks like this:
```xml
	<refsect1>
		<title>Description</title>
		<para>
			OpenSC obtains configuration data from the following sources in the following order
			<orderedlist>
				<listitem><para>
						command-line options
				</para></listitem>
				<listitem><para>
						environment variables
				</para></listitem>
				<listitem><para>
						Windows registry (if available)
				</para></listitem>
				<listitem><para>
						system-wide configuration file
						(<literal>@sysconfdir@/opensc.conf</literal>)
				</para></listitem>
			</orderedlist>
		</para>
```

**Pull request merged**